Mathlib/RingTheory/Ideal/Quotient/Basic.lean

```diff
@@ -85,23 +85,27 @@ lemma mk_singleton_self (x : R) [(Ideal.span {x}).IsTwoSided] : mk (Ideal.span {
 
 variable (I)
 
-instance noZeroDivisors [hI : I.IsPrime] : NoZeroDivisors (R ⧸ I) where
+instance noZeroDivisors [hI : I.IsCompletelyPrime] : NoZeroDivisors (R ⧸ I) where
     eq_zero_or_eq_zero_of_mul_eq_zero {a b} := Quotient.inductionOn₂' a b fun {_ _} hab =>
       (hI.mem_or_mem (eq_zero_iff_mem.1 hab)).elim (Or.inl ∘ eq_zero_iff_mem.2)
         (Or.inr ∘ eq_zero_iff_mem.2)
 
-instance isDomain [hI : I.IsPrime] : IsDomain (R ⧸ I) :=
+instance isDomain [hI : I.IsCompletelyPrime] : IsDomain (R ⧸ I) :=
   let _ := Quotient.nontrivial_iff.mpr hI.1
   NoZeroDivisors.to_isDomain _
 
-theorem isDomain_iff_prime : IsDomain (R ⧸ I) ↔ I.IsPrime := by
+theorem isDomain_iff_isCompletelyPrime : IsDomain (R ⧸ I) ↔ I.IsCompletelyPrime := by
```

Comment:
The corresponding statement for the new IsPrime says `R/I` is a [prime ring](https://en.wikipedia.org/wiki/Prime_ring) iff `I.IsPrime` (for two-sided `I`). A natural target is to show every group algebra K[G] over a field K is prime, which implies that Kaplansky's unit conjecture (disproved in general) for a particular K,G implies the zero divisors conjecture for that K,G. (screenshot from https://arxiv.org/abs/2302.11853)
<img width="976" height="352" alt="image" src="https://github.com/user-attachments/assets/17356f5b-442c-4b22-b9b9-64b85858bb29" />